web/modules/custom/dept_topics/css/topic-select.css

```diff
@@ -1,3 +1,15 @@
+/* Override Drupal core's inline styles that don't correctly calculate min-height for smaller devices */
+.ui-dialog {
+  top: 15% !important;
```

Comment:
I shifted the position using the jquery properties: https://github.com/dof-dss/nicsdru_dept/pull/541/files#diff-0469f9a5d52005e20af69565ea5530d5358d155049b6858ee97355e4cdb2205eR193